src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -551,12 +551,38 @@ public function validateDateFormat($attribute, $value, $parameters)
             return false;
         }
 
-        foreach ($parameters as $format) {
+        $formats = array_filter($parameters, fn (int|string $v) => $v !== 'loose');
+        $loose = $formats !== $parameters;
+
+        foreach ($formats as $format) {
             try {
-                $date = DateTime::createFromFormat('!'.$format, $value);
+                $date = DateTime::createFromFormat("!{$format}", $value);
+                $errors = DateTime::getLastErrors();
+
+                // Validates only that the specified format can be parsed.
+                //
+                // DateTime::getLastErrors() returns false when there are no errors or warnings in PHP 8.2 and later,
+                // and an array containing information about warnings or errors in PHP 8.1 and earlier.
+                // Therefore, to check if there is an error or warning,
+                // we need to first confirm that $errors is not false,
+                // and then make a judgement by looking at the contents of the array.
+                if (
+                    ! $date
+                    || (
+                        $errors !== false
+                        && ($errors['warning_count'] !== 0 || $errors['error_count'] !== 0)
+                    )
+                ) {
+                    continue;
+                }
```

Comment:
The above condition is quite complicated, can we describe it somehow? Or make a few conditions (to simplify)?

Reply:
I cut out the condition of whether there is an error or not!